Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -1170,6 +1176,10 @@ extension TabTrayController: MenuActionDelegate {
                 DispatchQueue.main.async {
                     self.closeTabsForCurrentTray()
                 }
+
+                if privateMode {
+                    LeanplumIntegration.sharedInstance.track(eventName: .closedPrivateTabs)
```

Comment:
Fair enough @jhugman - I'm meeting with Legal / Privacy to address this concern already, and note that product will make the ultimate decision on this. As we discussed, we are only tracking telemetry, and nothing about the contents of your private session.